### PR TITLE
chore(header): replace minimal headers with the epl related option

### PR DESCRIPTION
**Pull request by maximilianhuber**
Made mainly with the following `sed` script:
```
/\# Copying and distribution of this file, with or without modification/,/\# without any warranty./c\
# All rights reserved. This program and the accompanying materials are made\
# available under the terms of the Eclipse Distribution License v1.0 which\
# accompanies this distribution, and is available at\
# http://www.eclipse.org/org/documents/edl-v10.php
```

### CODE REVIEW

**Comment by mcjaeger** [comment]
I think it does not make sense for some of the file to be put under EPL-1.0, for example the notice file: why do we put the notice file under GPL? I am not sure if this is correct.

**Comment by bs-jokri** [comment]
Actually it is not the EPL but the EDL (Eclipse Documentation License, http://www.eclipse.org/org/documents/edl-v10.php), which is a BSD Style license and the most permissive Eclipse foundation accepts.

File: NOTICE-distribution [hunk 1]
```diff
@@ -331,10 +331,10 @@ THE SOFTWARE.
...
 --------------------------------------------------------------------------------
 
-Copying and distribution of this file, with or without modification,
-are permitted in any medium without royalty provided the copyright
-notice and this notice are preserved.  This file is offered as-is,
-without any warranty.
+All rights reserved. This program and the accompanying materials are made
+available under the terms of the Eclipse Distribution License v1.0 which
+accompanies this distribution, and is available at
+http://www.eclipse.org/org/documents/edl-v10.php
```
Comment (mcjaeger) [inline NOTICE-distribution]: maybe false hit of the script, pls consider removing.

**Comment by maximilianhuber** [comment]
The used sed-script was
```
/\# Copying and distribution of this file, with or without modification/,/\# without any warranty./c\
# All rights reserved. This configuration file is provided to you under the\
# terms and conditions of the Eclipse Distribution License v1.0 which\
# accompanies this distribution, and is available at\
# http://www.eclipse.org/org/documents/edl-v10.php
```
and applied via
```
$ find * -type f -exec sed -i -f ~/TNG/SCT/SW360/refactorHeader1.sed {} \;
```

**Pull request merged**